api/external/applications/applications.pb.gw.go

```diff
@@ -56,7 +56,10 @@ func local_request_ApplicationsService_GetServiceGroups_0(ctx context.Context, m
 	var protoReq ServiceGroupsReq
 	var metadata runtime.ServerMetadata
 
-	if err := runtime.PopulateQueryParameters(&protoReq, req.URL.Query(), filter_ApplicationsService_GetServiceGroups_0); err != nil {
+	if err := req.ParseForm(); err != nil {
```

Comment:
As best I can tell, this change is irrelevant to us, we don't use these functions.


In the gateway, we use the functions `RegisterTHINGHandlerFromEndpoint` (e.g., `RegisterApplicationsServiceHandlerFromEndpoint`) to register the handlers. If you follow the `RegisterTHINGHandlerFromEndpoint` implementation, you get to `RegisterTHINGHandlerClient` (e.g, `RegisterApplicationsServiceHandlerClient`, https://github.com/chef/automate/blob/28aa0d4a17681ec60a0574d13a568baa81a89612/api/external/applications/applications.pb.gw.go#L909-L920), and that method uses the functions that aren't prefixed with `local_`, e.g., like `request_ApplicationsService_GetServiceGroups_0` on line 40 above.